File: Javascript/A_NivelIniciante/Exercicio_1037.js
var input = require("./readFile.js").input;
var lines = input.split('\n');
var a = parseFloat(lines.shift())
if(a<0 || a>100){
    console.log("Fora de intervalo")
}else if(a>=0 && a<=25){
    console.log("Intervalo [0,25]");
}else if(a<=50){
    console.log("Intervalo (25,50]");
}else if(a<=75){
    console.log("Intervalo (50,75]")
}else if(a<=100){
    console.log("Intervalo (75,100]")
}

File: Python/A_NivelIniciante/Exercicio_1037.py
# -*- coding: cp1252 -*-
# -*- encoding: utf-8 -*-
#url do problema - https://www.urionlinejudge.com.br/judge/pt/problems/view/1037
A = float(input())          
if A<0 or A>100.0000:
    print("Fora do intervalo")
elif A>0 and A<=25.0000:
    print("Intervalo [0,25]")
elif A<=50.0000:
    print("Intervalo (25,50]")
elif A<=75.0000:
    print("Intervalo (50,75]")
elif A<=100.0000:
    print("Intervalo (75,100]")


